li-utils/src/main/java/com/linkedin/common/urn/MlFeatureUrn.java

```diff
@@ -0,0 +1,27 @@
+package com.linkedin.common.urn;
+
+public final class MlFeatureUrn extends Urn {
+
+  public static final String ENTITY_TYPE = "mlFeature";
+
+  private static final String CONTENT_FORMAT = "(%s,%s,%s)";
+
+  private final String featureName;
+
+  private final String dataType;
```

Comment:
Should this be an enum?

Reply:
Agreed! Added all the python data types initially - https://www.w3schools.com/python/python_datatypes.asp